dockerfiles/base/build.sh

```diff
@@ -6,7 +6,10 @@
 # http://www.eclipse.org/legal/epl-v10.html
 
 IMAGE_NAME="eclipse/che-base"
-. $(cd "$(dirname "$0")"; pwd)/../build.include
+base_dir=$(cd "$(dirname "$0")"; pwd)
+. $base_dir/../build.include
 
 init "$@"
 build
+
+source $base_dir/all_tests.sh
```

Comment:
we can't use source there because it doesn't work with "sh" (like dash)
we've to use "." to include content